README.md

```diff
@@ -327,9 +327,49 @@ interface IProviderCallback {
 
 ## Optional Flags
 
-You can enable the following optional flags:
+### Disable Injected Provider
 
-- disableInjectedProvider: disable displaying injected provider as an option (default: false)
+By default is set to `false` and Web3Connect always displays InjectedProvider as an option to the user if available. However you can disable it as an optional flag if you desire:
+
+#### React Button
+
+```javascript
+<Web3Connect.Button disableInjectedProvider />
+```
+
+#### Core Module
+
+```javascript
+const web3Connect = new Web3Connect.Core({ disableInjectedProvider: true });
+```
+
+### Cache Provider
+
+By default is set to `false` and Web3Connect will always require the user to choose a provider option before triggering the onConnect event. However you can enable caching the last chosen provider. Example:
+
+#### React Button
+
+```javascript
+<Web3Connect.Button cacheProvider />
+```
+
+#### Core Module
+
+```javascript
+const web3Connect = new Web3Connect.Core({ cacheProvider: true });
+```
+
+There are four options for this flag: `false`, `true`, `onclick` and `onload`. Here are the expected behaviors for each one of them:
```

Comment:
Where is the cache stored, is it worth documenting this, at least at a high level? Ignore me if it is somewhere else.

Reply:
It’s stored in localStorage, I will mention it in the README